### PR TITLE
make default value higher

### PR DESCRIPTION
**Pull request by 0xAshish**

### CODE REVIEW

File: contracts/staking/validatorShare/IValidatorShare.sol [hunk 1]
```diff
@@ -20,7 +20,7 @@ contract IValidatorShare is ERC20, Lockable, Ownable {
     uint256 public validatorRewards;
     uint256 public commissionRate;
     //# of checkpoints after which commission rate can be updated
-    uint256 public commissionCooldown = 1;
+    uint256 public commissionCooldown = 100; // epochs
```
Comment (jdkanani) [inline contracts/staking/validatorShare/IValidatorShare.sol:23]: No testcases changes?

Comment (0xAshish) [reply contracts/staking/validatorShare/IValidatorShare.sol:23]: WIP

Comment (jdkanani) [reply contracts/staking/validatorShare/IValidatorShare.sol:23]: Okay. Please notify once done!

**Comment by 0xAshish** [comment]
closing this PR need to rebase anyway.

**Pull request closed without merging**